setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: github-bot
+
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | Add the NodeSource Node.js repo
+      command: "bash -c 'curl -sL https://deb.nodesource.com/setup_6.x | bash -'"
+      tags: general
+
+    - name: General | APT Update and upgrade
+      apt: update_cache=yes upgrade=full
+      tags: general
+
+    - name: General | Install baseline packages
+      apt: name={{ item }} state=latest
+      with_items: "{{ baseline_packages }}"
+      tags: general
+
+    - name: General | Install required packages
+      apt: name={{ item }} update_cache=yes state=latest
+      with_items: "{{ packages }}"
+      tags: general
+
```

Comment:
@jbergstroem not sure I totally understand what you mean with running raw commands/avoid python. PTAL at https://github.com/nodejs/build/pull/469/commits/72f5ad9a710425c58b2970b6e43f01a6ca8b69bb seems to be what's needed on a local vagrant box I've got.. How would that compare to what you were thinking of?